securedrop_client/storage.py

```diff
@@ -164,15 +170,22 @@ def update_sources(remote_sources, local_sources, session, data_dir):
     session.commit()
 
 
-def update_files(remote_submissions, local_submissions, session, data_dir):
+def update_files(remote_submissions: List[SDKSubmission], local_submissions: List[File],
+                 session: Session, data_dir: str) -> None:
     __update_submissions(File, remote_submissions, local_submissions, session, data_dir)
 
 
-def update_messages(remote_submissions, local_submissions, session, data_dir):
+def update_messages(remote_submissions: List[SDKSubmission], local_submissions: List[Message],
+                    session: Session, data_dir: str) -> None:
     __update_submissions(Message, remote_submissions, local_submissions, session, data_dir)
 
 
-def __update_submissions(model, remote_submissions, local_submissions, session, data_dir):
+# Type for model should be Union[Type[File], Type[Message]] but
+# it is temporarily Any due to https://github.com/python/typing/issues/266
```

Comment:
I think you might be running `mypy` under python < 3.5 because I do not experience this because if I update the type annotation and run `pipenv run mypy --ignore-missing-imports securedrop_client`, this works.

Reply:
From the ticket in the comment, Python 3.5 releases below 3.5.3 will hit this issue, so I don't want to add this type hint given the weird traceback that devs will hit in their dev env when they run the unit tests

Reply:
OK to be fair the default version of python is gonna be at least 3.5.3 on most linux distros so it's pretty unlikely people hit this, added d29a27c4ba478b96ee207e5ec6a9646d7879cc7c